src/game/game.py

```diff
@@ -66,6 +66,10 @@ def move(grid, tile_to_move_arg):
     return new_grid
 
 
+def do_i_win(grid, started_grid):
```

Comment:
Can you find a better function name? It should be related to the grid.
If you return a boolean, you can start your fonction with `is_...` for example.